qml/TimeZonePage.qml

```diff
@@ -518,6 +547,12 @@ BasePage {
         }
     }
 
+    Timer {
+        interval: 1000
```

Comment:
This will refresh all the (currently instantiated) delegates of the list: as they don't display the second, we could set the interval to 30s, or even 60s, eventually, to make it lighter.